mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/wfs.servers.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Are you sure these files belong to /etc/georchestra/mapfishapp/js ?
I would have said /etc/georchestra/mapfishapp